pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/SmId.java

```diff
@@ -37,7 +37,7 @@ public class SmId {
     @ColumnName(DBConstants.SM_ID_PK)
     private Integer smIdPk;
     @ColumnName(DBConstants.DELETED)
-    private Boolean deleted;
+    private boolean deleted;
```

Comment:
I'm not sure it's safe to do this.  I see a bunch of `null` values in the `sm_id` field, and there's an existing constructor that attempts to unbox `Boolean -> boolean` on L54.  I think this change will result in an NPE somewhere.